checks/permissions.go

```diff
@@ -236,13 +252,19 @@ func calculateScore(result permissionCbData) int {
 	// Note: there are legitimate reasons to use some of the permissions like checks, deployments, etc.
 	// in CI/CD systems https://docs.travis-ci.com/user/github-oauth-scopes/.
 
-	if permissionIsPresent(result, "all") {
-		return checker.MinResultScore
-	}
-
 	// Start with a perfect score.
 	score := float32(checker.MaxResultScore)
 
+	// If no top level permissions are defined...
+	if permissionIsPresentInTopLevel(result, "all") {
+		if permissionIsPresentInRunLevel(result, "all") {
```

Comment:
Should this be `!permissionIsPresentInRunLevel`? The comments don't match the code.

Reply:
I'll update the comment. You're 100% right this is confusing! If permission "all" is present, it actually means no permissions were defined and by default all permissions are enabled.

Reply:
Maybe I'll rename the function instead... `permissionNotDefinedInRunLevel()`